extension/error.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  EINVAL: 'EINVAL', // validation code
+  EAUTH: 'EAUTH',
+  EINVALIDCREDENTIALS: 'EINVALIDCREDENTIALS',
+  EACCESS: 'EACCESS',
+  ENOTFOUND: 'ENOTFOUND',
+  EUNKNOWN: 'EUNKNOWN',
```

Comment:
EUNKNOWN should not be used from within steps. Therefore it can be removed here.

Reply:
removed